feature-utils/poly-look/src/react-components/overlays/sideSlider.jsx

```diff
@@ -0,0 +1,422 @@
+import React, { useEffect, useState, useRef } from "react";
+
+import "./sideSlider.css";
+
+let stPoz;
```

Comment:
I can move them to state, but it might give me some headaches or at least I might need to rethink some stuff. I've put them there because I've made the assumption that modules are preserved, so they are encapsulated. But looking at the dist code, that is not the case.

Reply:
Done.

Reply:
Also, naming nit... not easy to figure out what it means. Just use the fleshed-out name for the same low price of 0!